### ISSUE TITLE
Docker Test Button Failing

### ISSUE THREAD

**Issue by codygreen** [codygreen/lab-framework-demo#77]
When deploying an NGINX Plus container in the N1 lab guide, the Test button results in the following error message in the UI:

`Error: Failed API request: Cannot read properties of null (reading 'toString')`

Stack trace from the console:
```shell
API request failed: TypeError: Cannot read properties of null (reading 'toString')
    at eval (webpack-internal:///(rsc)/./node_modules/next/dist/server/lib/patch-fetch.js:272:82)
    at eval (webpack-internal:///(rsc)/./node_modules/next/dist/server/lib/trace/tracer.js:134:36)
    at NoopContextManager.with (webpack-internal:///(rsc)/./node_modules/next/dist/compiled/@opentelemetry/api/index.js:1:7062)
    at ContextAPI.with (webpack-internal:///(rsc)/./node_modules/next/dist/compiled/@opentelemetry/api/index.js:1:518)
    at NoopTracer.startActiveSpan (webpack-internal:///(rsc)/./node_modules/next/dist/compiled/@opentelemetry/api/index.js:1:18093)
    at ProxyTracer.startActiveSpan (webpack-internal:///(rsc)/./node_modules/next/dist/compiled/@opentelemetry/api/index.js:1:18854)
    at eval (webpack-internal:///(rsc)/./node_modules/next/dist/server/lib/trace/tracer.js:116:103)
    at NoopContextManager.with (webpack-internal:///(rsc)/./node_modules/next/dist/compiled/@opentelemetry/api/index.js:1:7062)
    at ContextAPI.with (webpack-internal:///(rsc)/./node_modules/next/dist/compiled/@opentelemetry/api/index.js:1:518)
    at NextTracerImpl.trace (webpack-internal:///(rsc)/./node_modules/next/dist/server/lib/trace/tracer.js:116:28)
    at patched (webpack-internal:///(rsc)/./node_modules/next/dist/server/lib/patch-fetch.js:233:41)
    at checkAPI (webpack-internal:///(action-browser)/./src/lib/check-api.js:64:32)
    at endpoint (webpack-internal:///(action-browser)/./node_modules/next/dist/build/webpack/loaders/next-flight-action-entry-loader.js?actions=%5B%5B%22%2Fworkspace%2Fsrc%2Flib%2Fcheck-api.js%22%2C%5B%22checkAPI%22%5D%5D%5D&__client_imported__=true!:8:17)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async /workspace/node_modules/next/dist/compiled/next-server/app-page.runtime.dev.js:39:418
    at async rw (/workspace/node_modules/next/dist/compiled/next-server/app-page.runtime.dev.js:38:7978)
    at async r6 (/workspace/node_modules/next/dist/compiled/next-server/app-page.runtime.dev.js:41:1256)
    at async doRender (/workspace/node_modules/next/dist/server/base-server.js:1438:30)
    at async cacheEntry.responseCache.get.routeKind (/workspace/node_modules/next/dist/server/base-server.js:1587:40)
    at async DevServer.renderToResponseWithComponentsImpl (/workspace/node_modules/next/dist/server/base-server.js:1507:28)
    at async DevServer.renderPageComponent (/workspace/node_modules/next/dist/server/base-server.js:1931:24)
    at async DevServer.renderToResponseImpl (/workspace/node_modules/next/dist/server/base-server.js:1969:32)
    at async DevServer.pipeImpl (/workspace/node_modules/next/dist/server/base-server.js:920:25)
    at async NextNodeServer.handleCatchallRenderRequest (/workspace/node_modules/next/dist/server/next-server.js:272:17)
    at async DevServer.handleRequestImpl (/workspace/node_modules/next/dist/server/base-server.js:816:17)
    at async /workspace/node_modules/next/dist/server/dev/next-dev-server.js:339:20
    at async Span.traceAsyncFn (/workspace/node_modules/next/dist/trace/trace.js:154:20)
    at async DevServer.handleRequest (/workspace/node_modules/next/dist/server/dev/next-dev-server.js:336:24)
    at async invokeRender (/workspace/node_modules/next/dist/server/lib/router-server.js:174:21)
    at async handleRequest (/workspace/node_modules/next/dist/server/lib/router-server.js:353:24)
    at async requestHandlerImpl (/workspace/node_modules/next/dist/server/lib/router-server.js:377:13)
    at async Server.requestListener (/workspace/node_modules/next/dist/server/lib/start-server.js:141:13)
 ⨯ src/lib/check-api.js (92:11) @ checkAPI
 ⨯ Error: Failed API request: Cannot read properties of null (reading 'toString')
    at checkAPI (./src/lib/check-api.js:86:15)
digest: "2278085431"
  90 |   } catch (error) {
  91 |     console.error('API request failed:', error);
> 92 |     throw new Error(`Failed API request: ${error.message}`);
     |           ^
  93 |   }
  94 | }
Debugger attached.
 POST /nginx-one.mdx 500 in 411ms
```

**Comment by codygreen** [comment]
fixed by #87

**Comment by codygreen** [comment]
code was removed, doing another PR.
